examples/cndpfwd/fwd.jsonc

```diff
@@ -122,17 +122,27 @@
     //   no-restapi - (O) Disable RestAPI support
     //   cli        - (O) Enable/Disable CLI supported
     //   mode       - (O) Mode type [drop | rx-only], tx-only, [lb | loopback], fwd, tx-only-rx,
-    //                    acl-strict, acl-permissive
+    //                    acl-strict, acl-permissive, [hyperscan | hs]
     //   uds_path   - (O) Path to unix domain socket to get xsk map fd
     "options": {
         "pkt_api": "xskdev",
         "no-metrics": false,
         "no-restapi": false,
         "cli": true,
         "mode": "drop",
+        // FIB rules <IP Address>,<Destination MAC>,<destination port>
         "l3fwd-fib-rules": [
             "198.18.0.0/24,02:00:01:02:03:04,1",
             "198.18.1.0/24,06:00:01:02:03:04,0"
+        ],
+        // The lport ID value (bits 16-23) is destination port, lower 16bits is unique ID value.
+        // If bits 31 is set then matching packets are dropped. See Hyperscan
+        // documentation for regex and flags values. <ID>:/<expression>/<flags>
+        // Any packets not matching a pattern below are dropped.
```

Comment:
Is this just true for the first rule with ID `0000`, the rest don't have bit 31 set?

Reply:
Correct, the rest are forwarding the packets to specific ports if they match a pattern.